client/app/dashboard/model/list/ModelList.js

```diff
@@ -373,7 +384,12 @@ angular.module('dashboard.Dashboard.Model.List', [
   
   $scope.getTotalServerItems = function() {
     var params = setupPagination();
-    GeneralModelService.count($scope.apiPath, params)
+    var countApiPath = $scope.apiPath;
+    if ($scope.action.options && $scope.action.options.searchCriteria) {
+       countApiPath = $scope.action.options.searchCriteria.custCountApi ? $scope.apiPath + '/' + $scope.action.options.searchCriteria.custCountApi : $scope.apiPath;
```

Comment:
I think this if clause is sort of strange.

Part of the condition is in the if expression and part of the condition is in the ternary operator.

I think it would be clearer written as:

`
if ($scope.action.options && $scope.action.options.searchCriteria && $scope.action.options.searchCriteria.custCountApi) {
 +       countApiPath = $scope.apiPath + '/' + $scope.action.options.searchCriteria.custCountApi;
`